docs/tutorials/UAV_tutorial.py

```diff
@@ -0,0 +1,272 @@
+# -------------------------
+# This part uses 2 Stone Soup detector type of classes:
+#
+# - :class:`~.CSVGroundTruthReader` - reads our CSV file which contains: timestamp,
+#   latitude, longitude, altitude and other miscellaneous data.
+# - :class:`~.LLAtoENUConverter` - this is a feeder, but it converts our lat, long, 
+#   alt data into Cartesian.
+#
+# The Cartesian data will be converted to Range, Bearing, Altitude later on.
+# More involved detector could:
+#
+# - Add clutter
+# - Handle :math:`P_d` -> Could be based on radial velocity
+# - Handle radar revisit times
+# - Add unknown number & multiple targets
+#
+from stonesoup.reader.generic import CSVGroundTruthReader
+
+ground_truth_reader = CSVGroundTruthReader(
+    path='UAV_Rot.csv',
+    state_vector_fields=['longitude', 'Vx m/s', 'latitude', 'Vy m/s', 'altitude (m)'],
+    time_field='time',
+    path_id_field='groupNb',
+)
+
+from stonesoup.feeder.geo import LLAtoENUConverter
+sensor_location = [-30.948, 50.297311666, 0]  # Radar position [long, lat, alt]
+ground_truth_reader = LLAtoENUConverter(ground_truth_reader, sensor_location, [0, 2, 4])
+
+# %%
+# Define Sensor, Platform and Detector
+# ------------------------------------
+# The sensor converts the Cartesian coordinates into range, bearing and elevation.
+# The sensor is then mounted onto a platform (stationary in this case)
+
+from stonesoup.platform.base import FixedPlatform
+from stonesoup.sensor.radar.radar import RadarRangeBearingElevation
+from stonesoup.simulator.platform import PlatformDetectionSimulator
+from stonesoup.types.state import State
+
+sensor = RadarRangeBearingElevation(
+    [0, 2, 4],
+    meas_covar,
+    6,
+)
+platform = FixedPlatform(
+    State([0, 0, 0, 0, 0, 0]),  # Sensor at reference point, zero velocity
+    [0, 2, 4],
+    sensors=[sensor]
+)
+
+# Create the detector and initialize it.
+detector = PlatformDetectionSimulator(ground_truth_reader, [platform])
+
+
+# %%
+# Setup Initiator and Deletor classes for the Tracker
+# ---------------------------------------------------
+# This is just an heuristic initiation:
+# Assume most of the deviation is caused by the Bearing measurement error.
+# This is then converted into x, y components using the target bearing. For z,
+# we simply use range*elev_std_dev (ignore any bearing or range related components).
+# Velocity covariances are just based on expected velocity range of targets.
+#
+# **NOTE** - The Extended Kalman filter can be very sensitive to the state
+# initiation. I tried using the default :class:`~.SimpleMeasurementInitiator` but it tended
+# diverge over the course of the track, especially when large bearing measurement
+# covariances.
+from stonesoup.types.state import GaussianState, State
+from stonesoup.types.update import GaussianStateUpdate
+from stonesoup.initiator.simple import SimpleMeasurementInitiator
+from stonesoup.types.track import Track
+from stonesoup.types.hypothesis import SingleHypothesis
+
```

Comment:
This is a nice example of building new class from an existing Stone Soup class. It might be worth spending a little time on the video showing how the principle of inheritance helps here. What would be required in building the class from scratch?